.travis.yml

```diff
@@ -5,6 +5,51 @@ notifications:
 
 jobs:
   include:
+    - name: "Backend cs-check"
+      stage: lint
+      language: php
+      php:
+        - 7.4.2
+      cache:
+        directories:
+          - vendor
+      before_script:
+          - cd backend
+          - composer install --no-interaction
```

Comment:
I guess we wouldn't need to install all the dependencies here, but we are caching `/backend/vendor` anyway.

Reply:
Or maybe we aren't, as the cache path is only `vendor`? And maybe we shouldn't even, after reading this https://github.com/travis-ci/travis-ci/issues/4579? Another post that suggests caching `$HOME/.composer/cache/files`: https://blog.wyrihaximus.net/2015/07/composer-cache-on-travis/.

Reply:
True, but we can handle correct caching in #681. So this is fine with me for now.